src/Bundle/DependencyInjection/WebpackExtension.php

```diff
@@ -72,7 +72,7 @@ private function getPlatformKey()
         if (strtoupper(substr(php_uname('s'), 0, 5)) === 'LINUX') {

```

Comment:
Optionally `PHP_OS` could be used, but not sure how stable that will be. http://php.net/manual/en/function.php-uname.php